src/components/ViewerTitle.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+import { ModelInfo } from '../constants';
+
+import '../../css/viewer_title.css';
+interface ViewerTitleProps {
+  modelInfo: ModelInfo;
+}
+
+const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
+  props: ViewerTitleProps
+): JSX.Element => {
+  const hasMetaData = (): boolean => {
+    for (const key in props.modelInfo) {
```

Comment:
if you make the change I suggested this would become 
```suggestion
    for (const key in props) {
```